### PR TITLE
fix(intake-formats) align with new ingest

### PR DESCRIPTION
**Pull request by gaelmuller**
Make sure tests remain compatible with future versions of ingest with stronger type enforcement.

### CODE REVIEW

File: Vectra/vectra_cognito_detect/tests/vectra_info.json [hunk 1]
```diff
@@ -6,42 +6,39 @@
...
         },
-        "observer":
-        {
-            "version":"6.7",
-            "name":"255.255.255.1",
-            "ip":"255.255.255.1"
+        "observer": {
+            "version": "6.7",
+            "name": "255.255.255.1",
+            "ip": "255.255.255.1"
```
Comment (yosradirgham) [inline Vectra/vectra_cognito_detect/tests/vectra_info.json:19]: Shouldn't this be something like this?
```suggestion
            "ip": ["255.255.255.1"]
```

**Review by yosradirgham — APPROVED** [review]
LGTM

File: Vectra/vectra_cognito_detect/tests/vectra_info.json (hunk 1 above)
Comment (PierrickV) [reply Vectra/vectra_cognito_detect/tests/vectra_info.json:19]: right https://www.elastic.co/guide/en/ecs/current/ecs-observer.html#field-observer-ip

Comment (gaelmuller) [reply Vectra/vectra_cognito_detect/tests/vectra_info.json:19]: You are right, but this is not the point of this PR

File: Vectra/vectra_cognito_detect/tests/vectra_lateral_movement.json [hunk 2]
```diff
@@ -42,7 +44,8 @@
         "id": 13295,
         "name": "SMB Brute-Force",
         "reason": "MORE_PROCESSING_REQUIRED",
-        "type": "smb_brute_force"
+        "type": "smb_brute_force",
+        "count": "295"
```
[the remark this answers is not in the document]
Comment (gaelmuller) [reply Vectra/vectra_cognito_detect/tests/vectra_lateral_movement.json:48]: It is currently defined as a keyword:

```
vectra.detection.count:
  name: vectra.detection.count
  type: keyword
  description: The number of attempts
```

**Review by PierrickV — APPROVED** [review]

**Pull request merged**